rmgpy/rmg/input.py

```diff
@@ -274,6 +274,21 @@ def quantumMechanics(
                                         onlyCyclics = onlyCyclics,
                                         maxRadicalNumber = maxRadicalNumber,
                                         )
+
+def mcnnEstimator(mode='thermochemistry'):
```

Comment:
Perhaps change the input argument to `thermochemistry=True`? That way, in the future we could potentially add `kinetics=True`, etc.

The if logic would also be a bit easier since you could just do `if thermochemistry:` and have Python automatically handle unsupported arguments.